src/main/java/teammates/common/util/Url.java

```diff
@@ -130,11 +81,11 @@ public String toString() {
     }
```

Comment:
It's a good idea, but if I modify the usages of `.toString()` and `.toAbsoluteString()` I'm making (many) more changes than I think this PR needs. Also, I don't think `getBaseUrl()` is needed at least for now.